update ES links

RafaelDavisH opened the pull request:
This PR is in replacement to the previous PR #40

raisedadead commented on default.hbs:
```diff
@@ -33,7 +33,7 @@
+            <h1 class="site-title">文章</h1>
             <h2 class="site-description">
                 This is a free, open source, no-ads place to cross-post your blog articles. Read about it
                 <a href="https://www.freecodecamp.org/forum/t/279929">here</a>.
```
We would want to translate and point to the Chinese forum link here instead?

miyaliu666 replied:
> We would want to translate and point to the Chinese forum link here instead? But that can be taken up separate of this pull-request.

Ah yes, the same sentence appeared several times, and I translated some of them and pointed to the Chinese forum https://chinese.freecodecamp.org/forum/t/freecodecamp/374 (an introduction to Chinese News). Not sure why I didn't translate this one:(
Should I cancel this PR and submit another one?

raisedadead replied:
You are able to edit the same file on the same branch, and this PR will automatically update.

raisedadead commented on error-404.hbs:
```diff
@@ -34,7 +34,7 @@ It's a good idea to keep this template as minimal as possible in terms of both f
+            <h1 class="site-title">欢迎来到 freeCodeCamp 社区</h1>
             <h2 class="site-description">
                 This is a free, open source, no-ads place to cross-post your blog articles. Read about it
                 <a href="https://www.freecodecamp.org/forum/t/279929">here</a>.
```
Same as https://github.com/freeCodeCamp/news-theme/pull/42/files#r301528968

miyaliu666 commented:
Please checked the updated.
I think we can delete social-row.hbs for now, as it encourages people to share on twitter/facebook. Later we may add: share on WeChat with scanning QR code. It may require registration on WeChat platform. I'll look into it. @raisedadead

naomi-lgbt commented:
Hey @RafaelDavisH! 

Is this PR separate from #40?

RafaelDavisH commented:
Yes. Disregard the last PR #40

moT01 reviewed (dismissed):
I noticed some of the titles are still in English - `Git Clone` and `Python Main` among others. I'm guessing that's intentional though. If so, LGTM 🎉

naomi-lgbt commented on locales/es.json:
```diff
@@ -29,35 +29,35 @@
+  "footer-col-3-1": "<a href='https://www.freecodecamp.org/espanol/news/diseno-web-responsive-como-hacer-que-un-sitio-web-se-vea-bien-en-telefonos-y-tabletas/'>Diseño Web Responsive</a>",
+  "footer-col-3-2": "<a href='https://www.freecodecamp.org/espanol/news/que-es-svg-explicacion-de-etiquetas-imagenes/'>¿Qué es un archivo SVG?</a>",
+  "footer-col-3-3": "<a href='https://www.freecodecamp.org/espanol/news/que-es-un-archivo-pdf-como-lo-abres/'>PDF Password Remover</a>",
+  "footer-col-3-4": "<a href='https://www.freecodecamp.org/news/what-is-a-pdf-file-and-how-do-you-open-it-solved/'>¿Qué es un Archivo PDF?</a>",
```
Should this link point to an `espanol` article?

RafaelDavisH replied:
Yes 🤦🏽

RafaelDavisH commented:
> I noticed some of the titles are still in English - `Git Clone` and `Python Main` among others. I'm guessing that's intentional though. If so, LGTM 🎉 

Yes, by convention we’re not translating some of the titles and others because those articles are not translated yet.

RafaelDavisH committed:
Update Spanish Interface strings

update: shorten links

add missing es url

Squash commits for Update ES Interface strings

scissorsneedfoodtoo reviewed (approved):
Tested locally and everything LGTM :+1: :+1: 

Thanks for making all these changes, @RafaelDavisH, and thanks everyone else for your thorough review.
